src/table.rs

```diff
@@ -53,6 +55,7 @@ impl fmt::Display for Table {
 }
 
 impl Table {
+    const ANTE_INCREASE_AMOUNT: i32 = 1;
```

Comment:
I expect we could just always leave this as 1, but is there any other reason we define this here, but ante_round_increase in the Table struct?

Reply:
Yeah I've update the doc strings. The struct one is like the period for when to update the ante, this is how much to increase the ante by.